crates/forge-runner/src/running.rs

```diff
@@ -286,6 +288,12 @@ pub fn run_test_case(
     let block_context = get_context(&forge_runtime).block_context.clone();
     let execution_resources = get_all_execution_resources(forge_runtime);
 
+    let max_steps = if case.max_steps.is_some() {
+        case.max_steps
+    } else {
+        runner_config.max_steps
+    };
+    check_max_steps_limit(&execution_resources, max_steps)?;
```

Comment:
Does this function check if the value of max steps is correct? name suggests it.

Reply:
yeah the name seems off